src/storageClients/CassandraTableStorageManager.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const Promise = require('bluebird');
```

Comment:
Let's stick with traditional `Promises` unless there's features from bluebird that we're using.